DesignSafe-CI/portal
# Task/des 2327 improve citation visibility

## Overview: ##
Improve Citation Visibility

## PR Status: ##

## Related Jira tickets: ##

* [DES-2327](https://jira.tacc.utexas.edu/browse/DES-2327)

## Summary of Changes: ##
The citation now appears before expanding a published entity and include options to download in various formats. Added download formats include RIS, BibTeX, and DataCite XML. The citation button was removed because the citation information is viewable upon loading the page for each entity. How names are listed in the citation has also changed. The changes apply to the Published view, and Publication Preview (in My Projects). 

## Testing Steps: ##
For each project type (Field Research, Experimental, Simulation, Hybrid Simulation, Other)
1. In the **Published tab**, find a published project, load the page and see the new citation format. 
2. Find that published project in the **My Project tab**. Load the project and go to Publication Preview. See the new citation format. There should also appear a green box saying "Published" next to the title.
3. In the **My Project tab**, find a project that has been published, but also has an unpublished entity. That unpublished entity should have a grey box saying "Unpublished". The citation title is different, and the citation itself if just a preview, without active links. 

## UI Photos:
![Screen Shot 2023-03-06 at 10 55 36 AM](https://user-images.githubusercontent.com/35277477/223180247-00f44b56-5a90-4c01-924d-f0a63b183fb3.png)
![Screen Shot 2023-03-06 at 10 53 37 AM](https://user-images.githubusercontent.com/35277477/223180251-cebe7eb2-1b3d-48a5-9cac-eab345c18d89.png)
![Screen Shot 2023-03-06 at 10 54 19 AM](https://user-images.githubusercontent.com/35277477/223180254-545f9f27-3ef4-481a-89af-dc056da6b044.png)
![Screen Shot 2023-03-06 at 10 51 37 AM](https://user-images.githubusercontent.com/35277477/223180256-1354ead5-d36d-442c-af64-265d4bf158a9.png)


## Notes: ##
My Projects does **not** show version number (Published does)
What has **NOT** changed: As the user works their way through the pipeline, there is no citation preview.
